src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -65,4 +65,19 @@ public static Association closestToValue(double value) {
         return Arrays.stream(values()).min(Comparator.comparingDouble(a -> Math.abs(a.value - value))).get();
     }
 
+    /**
+     * Fetch the {@link Association} convert value to association.
+     * No matter how friendly someone is as long as he is friendly he gets the ally association. The same goes for enemies.
+     * So values below 0 are enemies and above are allies.
+     *
+     * @param value The friendliness value
+     * @return      The {@link Association} with a friendliness value depending to the provided value
+     * @author IronException
+     */
+    public static Association convertValueToAssociation(double value) {
```

Comment:
Hmm, this method would be really confusing for someone who isn't familiar with the `Association` class, considering we already have:
* `byValue` (exact value for a `Association`)
* `closestToValue` (closest to exact value)
 and now, `convertValueToAssociation` (which doesn't really fit the naming scheme either!)

How about we rename `byValue` to `byExactValue` and rename `convertValueToAssociation` to `byValue`?

Reply:
I agree it is confusing. Somehow we need to fit the standards tho and make it possible to convert any positive value to an ally. (Idrk what the closestToValue method should be used for)

What did you mean was `byValue` for before the renaming?